spec/support/form_helpers.rb

```diff
@@ -337,6 +346,7 @@ def fill_in_activity_form(
         expect(page).to have_content policy_marker_nutrition
       end
     end
+    expect(page).to have_content fund_pillar if parent&.roda_identifier_compound&.include?("NF")
```

Comment:
Could you use one of the newer ways to identify that the parent is either the Newton fund or Newton-funded? Also check with @leeky about the inclusion check possibly being too generous. https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/770#discussion_r535067144